File: cours4/exo1.js
// variable hour en nombre
let hour


hour = Number(prompt('Entrez un nombre entre 0 et 24 : '))

if (hour >= 0 && hour < 6) {
    alert('Bonne nuit')
}  
else if(hour >= 6 && hour < 12) {
    alert('Passe une bonne matinée')
}  
else if (hour >= 12 && hour < 18) {
    alert('Passe une bonne après-midi')
}   
else if (hour >= 18 && hour < 24) {
    alert('Une bonne soirée')
} 
else {
    alert('MAIS IL ET FOU')
}
    

File: cours4/exo1.py
hour = int(input('Entrez un nombre entre 0 et 24 : '))

if hour >= 0 and hour < 6:
    print('Bonne nuit')
elif hour >= 6 and hour < 12:
    print('Passe une bonne matinée')
elif hour >= 12 and hour < 18:
    print('Passe une bonne après-midi')
elif hour >= 18 and hour < 24:
    print('Une bonne soirée')
else:
    print('MAIS IL ET FOU')